src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -72,7 +72,7 @@ public LocalizedHtmlString(string name, string value, bool isResourceNotFound, p
         public string Name { get; }
 
         /// <summary>
-        /// The string resource.
+        /// The string resource. It doesn't include arguments, even if they are passed in.
```

Comment:
Could you clarify how arguments could be passed into this property?

Reply:
@dougbu  how about the following two options?

It doesn't include string placeholder arguments, even if they were passed into the constructor.

It doesn't include string placeholder arguments, even if the instance was constructed with the arguments.

Reply:
:ok: I better understand what you're trying to say and agree the current comment is lacking. Suggest
```
The original resource string, without substitution of any constructor arguments.
```
or
```
The original resource string, prior to formatting with any constructor arguments.
```

(Hmm, I switched the order of "string resource" but that's unimportant. Up to you.)